collect_app/src/main/java/org/odk/collect/android/views/GeoImageButton.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2018 Yizheng Huang
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.odk.collect.android.views;
+
+import android.content.Context;
+import android.support.v7.widget.AppCompatImageButton;
+import android.util.AttributeSet;
+
+
+public class GeoImageButton extends AppCompatImageButton {
```

Comment:
I would change this name. It's now used only by GeoButtons but generally, it could be used in other places also right? Maybe in navigation buttons? SO I would recommend something more generall.